src/main/java/com/powsybl/openloadflow/util/Reports.java

```diff
@@ -697,9 +698,9 @@ public static void reportAcEmulationDisabledInWoodburyDcSecurityAnalysis(ReportN
     public static void reportContingencyActivePowerLossDistribution(ReportNode reportNode, double mismatch, double remaining) {
         reportNode.newReportNode()
                 .withMessageTemplate("contingencyActivePowerLossDistribution", "Contingency caused the loss of ${mismatch} MW injection: ${distributed} MW distributed, ${remaining} MW remaining.")
-                .withUntypedValue(MISMATCH, mismatch)
-                .withUntypedValue("distributed", mismatch - remaining)
-                .withUntypedValue("remaining", remaining)
+                .withUntypedValue(MISMATCH, String.format(Locale.UK, "%.2f", mismatch))
```

Comment:
This is not the right way to solve this issue because you lost the value type (double -> String) in the report model just for testing purpose.
This [PR](https://github.com/powsybl/powsybl-open-loadflow/pull/1187) will provide a better way to handle this.